test/integration/custom-routes/test/index.test.js

```diff
@@ -2563,7 +2563,7 @@ describe('Custom routes', () => {
     await fs.writeFile(nextConfigPath, nextConfigRestoreContent)
   })
 
-  describe('dev mode', () => {
+  describe.only('dev mode', () => {
```

Comment:
```suggestion
  describe('dev mode', () => {
```

Reply:
There's still an issue with rewrite resolving after the matchers change that needs investigation